test/SlowTests/Voron/Issues/RavenDB_10825.cs

```diff
@@ -37,7 +37,7 @@ public unsafe void Encryption_buffer_of_freed_scratch_page_must_not_affect_anoth
                 var cryptoTxState = ((IPagerLevelTransactionState)tx.LowLevelTransaction).CryptoPagerTransactionState.Single().Value;
 
                 // explicitly change the order of items in dictionary so we'll apply the buffer of overflow page before the already freed one
-                cryptoTxState.LoadedBuffers = cryptoTxState.LoadedBuffers.Reverse().ToDictionary(x => x.Key, x => x.Value);
+                cryptoTxState.ReverseBuffers();
```

Comment:
This is wrong. It _may_ work on a dictionary that is very small, but that is wrong. 
It certainly shouldn't be in the API

Reply:
This was the code that was written I have just hidden the dictionary

Reply:
I can delete the test if you want

Reply:
I don't think we want to delete a test. The test does this hack in order to confirm that something is working correctly but it doesn't need to use a dedicated method for that I think.

Reply:
if I expose the dictionary than I didn't do anything by hiding it.
I could add an expensive setter that will re-calculate the size of the buffers and if it is used only by one test than it should be okay.

Reply:
Then maybe let's rename the method `ReverseBuffers` to `ReverseBuffers_DebugOnly` and mark it as `internal` so it will be accessible in tests.

Reply:
This behavior is fine with the _test_, but it isn't fine in the API.